app/controllers/webhooks_controller.rb

```diff
@@ -9,9 +9,11 @@ def email
         return
       end
 
-      if p.from.address == 'postmaster@hmps.gsi.gov.uk'
-        logger.error "Sender ( postmaster@hmps.gsi.gov.uk ) detected. Skipping message.\n #{p.inspect}"
-        render text: 'Sender postmaster@hmps.gsi.gov.uk'
+      blacklist = %w(postmaster sendmaster webmaster no-reply noreply root)
+
+      if blacklist.include? p.from.address.split('@')[0]
+        logger.error "Blacklisted sender detected. Skipping message.\n #{p.inspect}"
```

Comment:
Do we want to dump the entire `inspect` here? If we get lots of mails, that might be noisy…